packages/atomic/src/components/commerce/facets/atomic-commerce-category-facet/atomic-commerce-category-facet.new.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  wrapInCommerceInterface,
+  playExecuteFirstSearch,
+  playKeepOnlyFirstFacetOfType,
+} from '@coveo/atomic/storybookUtils/commerce-interface-wrapper';
+import {parameters} from '@coveo/atomic/storybookUtils/common-meta-parameters';
+import {renderComponent} from '@coveo/atomic/storybookUtils/render-component';
+import type {Meta, StoryObj as Story} from '@storybook/web-components';
+import {html} from 'lit-html';
+
+const {play, decorator} = wrapInCommerceInterface({skipFirstSearch: true});
+
+const meta: Meta = {
+  component: 'atomic-commerce-category-facet',
+  title: 'Atomic-Commerce/CategoryFacet',
+  id: 'atomic-commerce-category-facet',
+  render: renderComponent,
+  decorators: [decorator],
+  parameters,
+  play,
+};
+
+export default meta;
+
+export const Default: Story = {
+  name: 'atomic-commerce-category-facet',
+  decorators: [
+    (_) => {
+      return html`<div id="code-root">
```

Comment:
Is the `<div id="code-root">` necessary here if there is already one in `renderComponent` ?

https://github.com/coveo/ui-kit/blob/KIT-3260/packages/atomic/storybookUtils/render-component.tsx#L31

Reply:
Yes, this part of `storybook-utils` does not end up being used in this case.